content/en/agent/cluster_agent/clusterchecksrunner.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Cluster Checks Runner
+kind: documentation
+further_reading:
```

Comment:
I think we should link the cluster checks runner blogpost here, it's going to be released very soon.
cc @cohenyair @davidmlentz in case you guys can provide the blogpost's link.

Reply:
Will set a reminder to link once the blog is live, checked in with Yair and he mentioned it'd either be live by today or this week 👍